apps/examples/app/components-v2/Image.tsx

```diff
@@ -0,0 +1,24 @@
+import { Box, Image, SafeAreaBox, Text } from '@ficus-ui/native';
+
+
+const ImageComponent = () => {
+  return (
+    <SafeAreaBox style={{ flex: 1 }}>
```

Comment:
```suggestion
    <SafeAreaBox flex={1}>
```
We can do that, right?